README.md

```diff
@@ -90,83 +91,149 @@ const requestResults = await interBTC.redeem.request(amount, btcAddress);
 
 At this point, one more more Vaults will send BTC to the address specified within 24 hours.
 
+### Creating an AccountId Instance
+
+Certain API calls require a parameter of type `AccountId`, which represents the Polkadot/Kusama account and can be instantiated as follows
+
+```ts
+import { AccountId } from "@polkadot/types/interfaces/runtime";
+
+const activeStakedRelayerId = await interBTC.api.createType(
+    "AccountId",
+    "5Ck5SLSHYac6WFt5UZRSsdJjwmpSZq85fd5TRNAdZQVzEAPT"
+);
+const feesEarnedByActiveStakedRelayer = await interBTC.stakedRelayer.getFeesEarned(
+    activeStakedRelayerId
+);
+```
+
 ### More Examples
 
-There are plenty more examples how to use this library. Best is to take a look at the integration tests: https://github.com/interlay/interbtc-api/tree/master/test/integration
+There are many examples in the integration tests, showing how to use this library. Take a look at them here: https://github.com/interlay/interbtc-api/tree/master/test/integration
 
 ## API Documentation
 
 Please check the documentation at https://docs.interlay.io/interbtc-api/#/classes/DefaultInterBtcApi
```

Comment:
This link is dead, but not sure what we should replace it with, the only one I found https://docs.interlay.io/#/developers/integration takes the content from this file I think, so maybe we should just remove it altogether?

Reply:
Good spot. Yes, let's remove this.